src/operator/contrib/ngraph.cc

```diff
@@ -198,6 +206,9 @@ bool NgraphSubgraphInferStorageType(const nnvm::NodeAttrs &attrs,
                                     mxnet::DispatchMode *dispatch_mode,
                                     std::vector<int> *in_attrs,
                                     std::vector<int> *out_attrs) {
+  CHECK(dispatch_mode != nullptr);
+  CHECK(in_attrs != nullptr);
+  CHECK(out_attrs != nullptr);
```

Comment:
Slightly off-topic, but can we add curly braces to the body of the `if` statement below?  Their absence led me to initially mis-read the code.